modules/making-open-source-more-inclusive.adoc

```diff
@@ -1,3 +1,16 @@
+// Module included in the following assemblies:
+//
+// * distr_tracing/distr_tracing_rn/distr-tracing-rn-2-0.adoc
+// * distr_tracing/distr_tracing_rn/distr-tracing-rn-2-1.adoc
+// * distr_tracing/distr_tracing_rn/distr-tracing-rn-2-2.adoc
+// * distr_tracing/distr_tracing_rn/distr-tracing-rn-2-3.adoc
+// * distr_tracing/distr_tracing_rn/distr-tracing-rn-2-4.adoc
+// * distr_tracing/distr_tracing_rn/distr-tracing-rn-2-5.adoc
+// * distr_tracing/distr_tracing_rn/distr-tracing-rn-2-6.adoc
+// * distr_tracing/distr_tracing_rn/distr-tracing-rn-2-7.adoc
+// * distr_tracing/distr_tracing_rn/distr-tracing-rn-2-8.adoc
+// * distr_tracing/distr_tracing_rn/distr-tracing-rn-2-9.adoc
+
 :_module-type: CONCEPT
```

Comment:
```suggestion
:_content-type: CONCEPT
```

Reply:
Thank you for spotting. Applying your suggestion and fixing this in the other files in this directory.

Reply:
Thank you for spotting this (even though it's not part of this PR). Applying.